lib/ts/recipe/emailverification/recipe.ts

```diff
@@ -164,4 +185,27 @@ export default class Recipe extends RecipeModule {
     isErrorFromThisRecipe = (err: any): err is STError => {
         return STError.isErrorFromSuperTokens(err) && err.fromRecipe === Recipe.RECIPE_ID;
     };
+
+    getEmailForUserId: GetEmailForUserIdFunc = async (userId, userContext) => {
+        if (this.config.getEmailForUserId !== undefined) {
+            const userRes = await this.config.getEmailForUserId(userId, userContext);
+            if (userRes.status !== "EMAIL_DOES_NOT_EXIST_ERROR") {
+                return userRes;
+            }
+        }
+
+        for (const getEmailForUserId of this.getEmailForUserIdFuncsFromOtherRecipes) {
+            const res = await getEmailForUserId(userId, userContext);
+            if (res.status !== "EMAIL_DOES_NOT_EXIST_ERROR") {
```

Comment:
Shouldn't this be `!== "UNKNOWN_USER_ID_ERROR"`

Reply:
weird, I thought I fixed these before... :/ it's fixed now.